src/components/layout/SideBar.jsx

```diff
@@ -13,6 +13,12 @@ const { Sider } = Layout;
 
 const SideBar = props => {
 	const { context, dispatch } = useContext(Context);
+	
+	const [winWidth, setWinWidth] = useState(window.innerWidth);
+	function change(wid){ setWinWidth(wid) }
```

Comment:
This function is unnecessary, you can just use setWinWidth everywhere you use change